src/game/control.c

```diff
@@ -305,7 +305,7 @@ int32_t ControlPhase(int32_t nframes, GAMEFLOW_LEVEL_TYPE level_type)
 
         CalculateCamera();
         Sound_UpdateEffects();
-        g_GameInfo.stats.timer++;
+        g_GameInfo.current->stats.timer++;
```

Comment:
This is wrong. It needs to be `g_GameInfo.current[g_CurrentLevel].stats`, otherwise it always refers to the first level which afair is title screen.